apps/condo/domains/settings/schema/MobileFeatureConfig.test.js

```diff
@@ -0,0 +1,195 @@
+                    })
+                    await createTestOrganizationEmployee(admin, organization, client.user, role)
+
+                    const [obj, attrs] = await createTestMobileFeatureConfig(client, organization)
+
+                    expect(obj.id).toMatch(UUID_RE)
+                    expect(obj.dv).toEqual(1)
+                    expect(obj.sender).toEqual(attrs.sender)
+                    expect(obj.createdBy).toEqual(expect.objectContaining({ id: client.user.id }))
+                })
+
+                test('can update', async () => {
+                    const client = await makeClientWithNewRegisteredAndLoggedInUser()
+                    const [organization] = await createTestOrganization(admin)
+
+                    const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                        canManageOrganization: true,
+                    })
+                    await createTestOrganizationEmployee(admin, organization, client.user, role)
+
+                    const [objCreated] = await createTestMobileFeatureConfig(admin, organization)
+
+                    const [obj, attrs] = await updateTestMobileFeatureConfig(client, objCreated.id)
+
+                    expect(obj.id).toMatch(UUID_RE)
+                    expect(obj.dv).toEqual(1)
+                    expect(obj.sender).toEqual(attrs.sender)
+                    expect(obj.v).toEqual(2)
+                    expect(obj.updatedBy).toEqual(expect.objectContaining({ id: client.user.id }))
+                })
+
+                test('can\'t delete', async () => {
+                    const client = await makeClientWithNewRegisteredAndLoggedInUser()
+
+                    await expectToThrowAccessDeniedErrorToObj(async () => {
+                        await MobileFeatureConfig.delete(client, 'id')
+                    })
+                })
+
+                test('can read', async () => {
+                    const client = await makeClientWithNewRegisteredAndLoggedInUser()
+                    const [organization] = await createTestOrganization(admin)
+
+                    const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                        canManageOrganization: true,
+                    })
+                    await createTestOrganizationEmployee(admin, organization, client.user, role)
+
+                    const [objCreated] = await createTestMobileFeatureConfig(admin, organization)
+
+                    const objs = await MobileFeatureConfig.getAll(client, {}, { sortBy: ['updatedAt_DESC'] })
+
+                    expect(objs).toHaveLength(1)
+                    expect(objs[0]).toMatchObject({
+                        id: objCreated.id,
+                    })
+                })
+            })
+        })
+
+        describe('Validation tests', () => {
+            test('TICKET_SUBMITTING_PHONES_NOT_CONFIGURED', async () => {
+                await catchErrorFrom(async () => {
```

Comment:
seems to have recently fixed a function to catch GQLError

Reply:
Agree that it looks much better. (See example https://github.com/open-condo-software/condo/blob/e301d082f0999be098b61a1f1358b0d739864109/apps/condo/domains/miniapp/schema/B2BAppPermission.test.js#L364)
We also immediately make it clear that we expect `GQLError` and not some kind of error

Reply:
ok changed it